controllers/contentlibrary/v1alpha2/clustercontentlibraryitem/clustercontentlibraryitem_controller.go

```diff
@@ -245,13 +245,31 @@ func (r *Reconciler) setUpCVMIFromCCLItem(ctx *context.ClusterContentLibraryItem
 		return err
 	}
 
+	// Setting the label Key Prefix on the basis of
+	// WCP_TKG_Multiple_CL FSS is enabled or not
+
+	labelKeyPrefix := "type.services.vmware.com"
+	if pkgconfig.FromContext(ctx).Features.TKGMultipleCL {
+		labelKeyPrefix = "services.supervisor.vmware.com"
+	}
+
 	if cvmi.Labels == nil {
 		cvmi.Labels = make(map[string]string)
+	} else {
+		// Remove the labels from CVMI which are not a part of CCLItem
```

Comment:
This is dangerous as we should be adding labels to the VMI and CVMI resources for purposes of discoverability. It's part of the API spec in fact (see the Go documentation for `status.capabilities` and guest information. Now, we do not currently do that, but it's part of the API spec and on the deck to be implemented. Won't this change remove those labels?

Also, should this change not also be in place (or not) for non-cluster scoped VMIs?